acceptance/acceptance_test.go

```diff
@@ -747,7 +769,7 @@ func createBuilder(t *testing.T, runImageMirror string) string {
 			t.Fatal("LIFECYCLE_PATH must be an absolute path")
 		}
 		t.Logf("Adding lifecycle path '%s' to builder config", lifecyclePath)
-		_, err = builderConfigFile.Write([]byte(fmt.Sprintf("uri = \"%s\"\n", lifecyclePath)))
+		_, err = builderConfigFile.Write([]byte(fmt.Sprintf("uri = \"%s\"\n", strings.ReplaceAll(lifecyclePath, `\`, `\\`))))
```

Comment:
what is the purpose of this? I think I'm missing something

Reply:
This is necessary for TOML with Windows paths.